testing/src/Replay/Exception/NonDeterministicWorkflowException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Temporal\Testing\Replay\Exception;
+
+final class NonDeterministicWorkflowException extends ReplayerException
+{
+}
```

Comment:
It's worth having a docstring here that links to  https://docs.temporal.io/workflows/#deterministic-constraints